sdk/aqueduct/flow_run.py

```diff
@@ -64,6 +68,26 @@ def describe(self) -> None:
             assert param_op.spec.param is not None, "Artifact is not a parameter."
             print("* " + param_op.name + ": " + param_op.spec.param.val)
 
+    def artifact(
+        self, name: str
+    ) -> Optional[Union[TableArtifact, MetricArtifact, CheckArtifact, ParamArtifact]]:
+        """Gets the Artifact from the flow based on artifact_name"""
+        flow_run_dag = self._dag
+        artifact_from_dag = flow_run_dag.get_artifacts_by_name(name)
+
+        if artifact_from_dag is None:
+            raise ArtifactNotFoundException("The artifact name provided does not exist.")
+        elif get_artifact_type(artifact_from_dag) is ArtifactType.TABLE:
+            return TableArtifact(self._api_client, self._dag, artifact_from_dag.id, True)
+        elif get_artifact_type(artifact_from_dag) is ArtifactType.NUMBER:
+            return MetricArtifact(self._api_client, self._dag, artifact_from_dag.id, True)
+        elif get_artifact_type(artifact_from_dag) is ArtifactType.BOOL:
+            return CheckArtifact(self._api_client, self._dag, artifact_from_dag.id, True)
+        elif get_artifact_type(artifact_from_dag) is ArtifactType.PARAM:
+            return ParamArtifact(self._api_client, self._dag, artifact_from_dag.id, True)
```

Comment:
nit: for each of the artifact constructors, "True" is not a super enlightening argument value, so I typically do eg:
```
ParamArtifact(self._api_client, self._dag, artifact_from_dag.id, from_flow_run=True)
```
especially if its one of the last few arguments.